x-pack/plugins/apm/public/components/alerting/chart_preview/index.tsx

```diff
@@ -31,13 +31,15 @@ interface ChartPreviewProps {
   data?: Coordinate[];
   threshold: number;
   uiSettings?: IUiSettingsClient;
+  multiData?: Array<{ name: string; data: Coordinate[] }>;
```

Comment:
I don't think we should have both `data` and `multiData`. Lets instead rename `data` to `series` which is always an array:

```suggestion
  threshold: number;
  uiSettings?: IUiSettingsClient;
  series?: Array<{ name: string; data: Coordinate[] }>;
```